storybook/stories/Examples/Pie/PieWithNeedle.stories.tsx

```diff
@@ -70,10 +70,10 @@ export const PieWithNeedle = {
 };
 
 export const PieWithPatterns = {
-  render: (_args: Record<string, any>) => {
+  render: (args: Record<string, any>) => {
     return (
       <ResponsiveContainer width="100%" height={500}>
-        <PieChart>
+        <PieChart {...args}>
```

Comment:
The args should be spread to the component whose example this is. 
This example is about the Pie, not about the PieChart. 

There should be separate examples for each. 
For this example, the props should be spread to the Pie, and the values should be the default args of the story